ethcore/builtin/Cargo.toml

```diff
@@ -7,17 +7,18 @@ edition = "2018"
 
 [dependencies]
 bn = { git = "https://github.com/paritytech/bn", default-features = false }
+byteorder = "1.3.2"
+eip-152 = { path = "../../util/EIP-152" }
```

Comment:
not an issue, but I believe that all deps were supposed to be in order :)
if you're using vim, it is `ctrl-v` + `:sort`

Reply:
yeah, I did before but I must have messed it up when sorting out merge conflicts.